server/src/gql/resolver.py

```diff
@@ -72,3 +72,66 @@ def resolve_add_new_mates(
     family.mate_ids += [new_mate.id for new_mate in new_mates]
     family.save()
     return new_mates
+
+
+@mutation.field("assignMatesToFamilies")
+def resolve_assign_mates_to_families(
```

Comment:
For this method, I think the only real sanity checks against the user input we have to perform are "does this mate_id exist and belong to the user" and "does this family_id exist and belong to the user". I think everything else the user tries to do with this is fair game.

Reply:
To make it cleaner for the backend, imo we should change the gql to have `MateAssignmentInput` to just contain a list of `fromFamilyId, toFamilyId, mateId` since all of that information is readily available in the client when making this call, would make the logic here significantly easier, and could be re-used for general mate reassignment.